### ISSUE TITLE
Aborted( core dumped)

### ISSUE THREAD

**Issue by vkrgowtham** [RasaHQ/rasa_nlu#502]
**rasa NLU version** (e.g. `0.7.3`): 0.9.1

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...): mitie

**Operating system** (windows, osx, ...): ubuntu 16.04 LTS

**Issue** : Please look at the following error. I was able to train a dataset of around 50 examples with the same pipeline and installation, but with different training data I'm getting this error

overall accuracy: 1
Training time: 17324 seconds.
df.number_of_classes(): 17

INFO:rasa_nlu.model:Finished training component.
python: src/mitie.cpp:87: const T& {anonymous}::checked_cast(const void*) [with T = mitie::named_entity_extractor]: Assertion `ptr' failed.
Aborted (core dumped)

**Comment by wrathagom** [comment]
Have you tried the solutions mentioned in #239, which seems to be this exact same error?

Is it possible to share your hardware info: CPU, RAM, Docker or not? Also it may be helpful if you could provide your training data in a gist or privately in an e-mail to @tmbo and myself to see if anyone here can reproduce this error.

**Comment by vkrgowtham** [comment]
I have used an aws instance C4.4xlarge(30GB Ram, 16 cores) to overcome the memory problem. But the issue seems to persist.

**Comment by wrathagom** [comment]
Can you provide your complete config file and training data? Do you need MITIE for a particular reason?

**Comment by vkrgowtham** [comment]
mitie+ sklearn is not providing greater accuracy, so i have tried with mite pipeline. Config file is just the  default one with mitie pipeline.

**Comment by wrathagom** [comment]
and your training data? either in a gist or e-mail. Did you modify the config file for number of threads when you ran it on the EC2 instance?

**Comment by tmbo** [comment]
closed due to inactivity
